transformations/syntactically_diverse_paraphrase/transformation.py

```diff
@@ -0,0 +1,55 @@
+import benepar
+import spacy
+import nltk
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+from transformations.syntactically_diverse_paraphrase.sowreap.parse_utils import (
+    Sentence,
+)
+from transformations.syntactically_diverse_paraphrase.sowreap.reap_utils import (
+    reapModel,
+)
+from transformations.syntactically_diverse_paraphrase.sowreap.sow_utils import (
+    sowModel,
+)
+
+
+class ParaphraseSowReap(SentenceOperation):
+    tasks = [TaskType.TEXT_TO_TEXT_GENERATION]
+    languages = ["en"]
+
+    def __init__(self, seed=0, max_outputs=1):
+        super().__init__(seed, max_outputs=max_outputs)
+        self.sow = sowModel("tanyagoyal/paraphrase-sow", max_outputs)
```

Comment:
Can you add a comment here or in the README that provides details on this model? I think that it is a model directly hosted in huggingface database but it would be worth mentioning it.

Reply:
Can you elaborate on this? What kind of information about the model?

Reply:
I think that it would be good to have a paragraph detailing each model in the Readme.
Size of the model, date of training, on which data, how, who is maintaining the model, what does it do, ...